.github/workflows/wiki-sync.yml

```diff
@@ -0,0 +1,64 @@
+---
+name: Docs/Wiki Sync
+
+on:
+  push:
+    branches:
+      - dev
```

Comment:
The reason I was thinking `dev` is because these are dev docs and devs will work out of `dev`. If we change our branching strategy or guidance in `dev`, then we want people to see that right away. The public docs are built out of `main`.